python/mxnet/onnx/mx2onnx/_export_onnx.py

```diff
@@ -65,7 +65,7 @@ def __init__(self):
         self.output_tensors = []
 
     @staticmethod
-    def register(op_name, opset_version=11):
+    def register(op_name, opset_version=12):
```

Comment:
Why would we change lowest supported version from 11 to 12?

Reply:
I think we support onnx >= 1.7 and 1.7 is actually opset 12.